bfdd/bfd.c

```diff
@@ -31,6 +31,8 @@
 
 #include "bfd.h"
 
+extern struct zebra_privs_t bfdd_privs;
```

Comment:
Please move this declaration to the `bfd.h` header (already mentioned this in another commit).